R/fa.R

```diff
@@ -44,6 +51,8 @@ fa <- function(name,
     svg <- fa_tbl[fa_tbl$full_name == name, ][1, "svg"]
   } else if (name %in% fa_tbl$name) {
     svg <- fa_tbl[fa_tbl$name == name, ][1, "svg"]
+  } else if (name %in% fa_tbl$v4_name) {
+    svg <- fa_tbl[fa_tbl$v4_name == name, ][1, "svg"]
```

Comment:
This should also emit a message that tells the user that they're using an old v4 name, and which v5 name they should use instead.

Reply:
Done in both functions. Thanks!